CodeEdit/AppDelegate.swift

```diff
@@ -233,6 +221,12 @@ final class AppDelegate: NSObject, NSApplicationDelegate, ObservableObject { //
         }
     }
 
+    /// Enable window size restoring on app relaunch after quitting.
+    static func enableWindowSizeSaveOnQuit() {
+        // This enables window restoring on normal quit (instead of only on force-quit).
+        UserDefaults.standard.setValue(true, forKey: "NSQuitAlwaysKeepsWindows")
+    }
```

Comment:
What is the reason this function `static`?

Reply:
I marked it as static, as it doesn't really "belong" to an instance of AppDelegate here. I can remove it if you prefer